stockflux-securities-master/src/components/Alert.css

```diff
@@ -0,0 +1,11 @@
+.securities-message {
+  font-size: var(--font-xxxlarge);
+}
+
+.error {
+  color: red;
```

Comment:
Did you mean to change this colour? It should probably refer to a variable. You probably don't want the browser default `red` colour.

Reply:
I didn't change this color but yes I see that it should probaly be a variable. As it stands there is no css variable defined for error text but I can create one.